haas/test_common.py

```diff
@@ -254,11 +257,12 @@ def site_layout():
         api.switch_register(**switch)
 
     for node in layout['nodes']:
-        api.node_register(node['name'],obm=node['obm'])
+        api.node_register(node['name'], obm=node['obm'])
         for nic in node['nics']:
             api.node_register_nic(node['name'], nic['name'], nic['mac'])
             api.switch_register_port(nic['switch'], nic['port'])
-            api.port_connect_nic(nic['switch'], nic['port'], node['name'], nic['name'])
+            api.port_connect_nic(nic['switch'], nic['port'], node[
+                                 'name'], nic['name'])
```

Comment:
I used autopep8 to do all the whitespace stuff.  I thought I did all the line splitting, but I assure you that one was not me (at least a sober me).  I will comb through the code to see if there are any other line splits like this that are silly.